### PR TITLE
Docker container actions and service containers only work on 'ubuntu-latest'

### PR DESCRIPTION
**Pull request by dsame**
replaced "`ubuntu-latest` image" with "ubuntu images" in the phrase 

```
Note: If your workflows use Docker container actions or service containers, then you must use a Linux runner:

If you are using GitHub-hosted runners, you must use the ubuntu-latest runner.
```

See https://docs.github.com/en/free-pro-team@latest/actions/guides/about-service-containers#about-service-containers
Pay attention the is a reused data and blue block is to be changed across the whole documentation set.

The related issue: https://github.com/actions/virtual-environments/issues/1624

### Check off the following:
- [ ] All of the tests are passing.
- [ ] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by AlenaSviridenko** [comment]
LGTM

some context: Docker-Moby recently added support of Ubuntu-20.04 so now it is correct to say that Docker container actions or service containers can be used on all Ubuntu images.

**Comment by janiceilene** [comment]
Thanks for opening a PR @dsame! I'll get this triaged for review ✨

**Comment by marleyjustin08** [comment]
###

**Review by marleyjustin08 — APPROVED** [review]

**Review by marleyjustin08 — APPROVED** [review]

**Comment by marleyjustin08** [comment]
__

File: data/reusables/github-actions/docker-container-os-support.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 **Note:** If your workflows use Docker container actions or service containers, then you must use a Linux runner:
 
-* If you are using {% data variables.product.prodname_dotcom %}-hosted runners, you must use the `ubuntu-latest` runner.
+* If you are using {% data variables.product.prodname_dotcom %}-hosted runners, you must use ubuntu runners.
```
Comment (lucascosti) [inline data/reusables/github-actions/docker-container-os-support.md:5]: Slight tweak to capitalize 'Ubuntu' and to refer to a single runner. 

```suggestion
* If you are using {% data variables.product.prodname_dotcom %}-hosted runners, you must use an Ubuntu runner.
```

**Review by lucascosti — APPROVED** [review]
Thanks a lot for this PR, @dsame! 🙇‍♂️

I might a slight change, and when the tests pass, I'll merge this in. 🚀

**Pull request merged**